modules/ivts/dev.galasa.ivts/dev.galasa.ivts.core/build.gradle

```diff
@@ -0,0 +1,60 @@
+
+// This section tells gradle which gradle plugins to use to build this project.
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'dev.galasa.tests' version '0.40.0'
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+// This section tells gradle where it should look for any dependencies
+repositories {
+    mavenLocal()
+    mavenCentral()
+    // To use the bleeding edge version of galasa's obr plugin, use the development obr
+    maven {
+       url 'https://development.galasa.dev/main/maven-repo/obr'
+    }
+
+}
+
+// Set the variables which will control what the built OSGi bundle will be called
+// and the name it will be published under in the maven repository.
+group = 'dev.galasa.ivts'
+version = '0.0.1-SNAPSHOT'
+description = 'Galasa Core Manager IVTs'
+
+// What are the dependencies of the test code ? 
+// When more managers and dependencies are added, this list will need to grow.
+dependencies {
+    implementation platform('dev.galasa:galasa-bom:0.40.0')
```

Comment:
Need set-version.sh to keep this up to date.

Reply:
I will be doing all version related things when the POC works and I'm ready to deliver this PR